File: Pattern/Python/Pattern19.py
n = int(input("Enter number of rows: "))

for i in range(n):
    # Left stars
    for j in range(n - i):
        print("*", end="")

    # Spaces
    for k in range(i * 2 + 1):
        print(" ", end="")

    # Right stars
    for j in range(n - i):
        print("*", end="")

    print()

for i in range(n - 1, -1, -1):
    # Left stars
    for j in range(n - i):
        print("*", end="")

    # Spaces
    for k in range(i * 2 + 1):
        print(" ", end="")

    # Right stars
    for j in range(n - i):
        print("*", end="")

    print()

File: Pattern/Java/Pattern19.java
import java.util.*;

public class Pattern19 {
    public static void main(String[] args){

        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter number of rows: ");
        int n=sc.nextInt();

        for(int i=0; i<n; i++){

            for(int j = n-i; j > 0; j--){
                System.out.print("*");
            }

            for(int k=0; k<=i*2; k++){
                System.out.print(" ");
            }
            

            for(int j = n-i; j > 0; j--){
                System.out.print("*");
            }

            System.err.println();
        }

        for(int i=n-1; i>=0; i--){

            for(int j = n-i; j > 0; j--){
                System.out.print("*");
            }

            for(int k=0; k<=i*2; k++){
                System.out.print(" ");
            }
            

            for(int j = n-i; j > 0; j--){
                System.out.print("*");
            }

            System.err.println();
        }

        sc.close();
        
    }
    
}
